exchange/auction.go

```diff
@@ -17,14 +17,21 @@ import (
 	"github.com/prebid/prebid-server/prebid_cache_client"
 )
 
+const (
+	DebugOverrideHeader string = "x-pbs-debug-override"
+)
+
 type DebugLog struct {
-	Enabled     bool
-	CacheType   prebid_cache_client.PayloadType
-	Data        DebugData
-	TTL         int64
-	CacheKey    string
-	CacheString string
-	Regexp      *regexp.Regexp
+	Enabled       bool
+	CacheType     prebid_cache_client.PayloadType
+	Data          DebugData
+	TTL           int64
+	CacheKey      string
+	CacheString   string
+	Regexp        *regexp.Regexp
+	DebugOverride bool
+	//little optimization, it stores value of debugLog.Enabled || debugLog.DebugOverride
+	DebugConsolidated bool
```

Comment:
Consider renaming to avoid the comment, perhaps `DebugEnabledOrOverriden`. There's also a chance for deviation if `debugLog.Enabled` or `debugLog.DebugOverride` are changed. Do you think this would be better served as a method on the `debugLog` struct?

I'm not concerned with the runtime performance of or-ing two booleans.

Reply:
Renamed. 
`There's also a chance for deviation if debugLog.Enabled or debugLog.DebugOverride are changed` -> DebugEnabledOrOverriden calculates per request. the only one place where we set value to this field is video_endpoint:
`debugLog.DebugEnabledOrOverridden = debugLog.Enabled || debugLog.DebugOverride`
I don't see any benefits to implement a method on the debugLog struct.

Originally there was `or` between two variables, but then it was extracted to separate field to optimize code.